src/subcommands/chain.rs

```diff
@@ -1,4 +1,8 @@
 pub fn run_chain(chain_args: Vec<String>) -> Result<(), failure::Error> {
     crate::emit!(cmd.chain, { "args": chain_args });
-    crate::command::run_cmd("oasis-chain", chain_args, crate::command::Verbosity::Normal)
+    crate::command::run_cmd(
+        "oasis-chain",
+        &chain_args.iter().map(|a| a.as_str()).collect::<Vec<_>>(),
```

Comment:
Nick is on this one :)

Reply:
that actually can't be changed. String is different from &str and making `run_cmd` take both makes things slightly more complicated. If something else ends up doing this, I'll change it, but not just yet.